README.md

```diff
@@ -4,47 +4,86 @@
 Placeholder for the Metadata Query Service
 
 
+# Query Service
+[![Test Coverage](https://codecov.io/gh/HumanCellAtlas/query-service/branch/master/graph/badge.svg)](https://codecov.io/gh/HumanCellAtlas/query-service)
+[![Master Deployment Status](https://allspark.dev.data.humancellatlas.org/HumanCellAtlas/query-service/badges/master/pipeline.svg)](https://allspark.dev.data.humancellatlas.org/HumanCellAtlas/query-service/commits/master)
+
+######TODO add these links
+[![Production Health Check](health check badge SVG)](URL to service)
+
+
+## Overview
+
+The Query Service (QS) provides an interface for scientists and developers to query metadata associated with gene 
+expression data stored in the [Human Cell Atlas](https://staging.data.humancellatlas.org/) [Data Coordination
+Platform](https://www.humancellatlas.org/data-sharing) (DCP). Metadata from the
+[DCP Data Store](https://github.com/HumanCellAtlas/data-store) are indexed and stored in an [AWS Aurora PostgreSQL](add_url) 
+database, query requests are  processed by [AWS Lambda](https://aws.amazon.com/lambda/)
+queries submitted through the `/query` endpoint directly return the data while the results of longer running queries 
+(`/long-running-query`) are stored in a [Amazon S3](https://aws.amazon.com/s3/) bucket. The service exposes a 
+[REST API](https://query.staging.data.humancellatlas.org) for querying and retrieving
+data.   
+### Components
+
+[LucidChart Architecture Diagram](https://www.lucidchart.com/invitations/accept/02e38662-6da1-48ad-87ac-0355bb3a03d8)
+![alt text](query_arch.svg)
+
+
+#### Query API
+
+The REST API is a [Chalice](https://github.com/aws/chalice) app that adopts [Swagger/OpenAPI](https://swagger.io/)'s
+approach to specification driven development and leverages [Connexion](https://github.com/zalando/connexion) for
+input parameter validation. The Chalice app is deployed with [Amazon API Gateway](https://aws.amazon.com/api-gateway/)
+and [AWS Lambda](https://aws.amazon.com/lambda/). The full API documentation can be found
+[here](https://query.staging.data.humancellatlas.org).
+
+
+#### Load Data
+
+The query service is subscribed to all updates to the [DCP Data Store (DSS)](https://github.com/HumanCellAtlas/data-store). 
+When data is added to the DSS a webhook containing the bundle id is sent to the Query API. The bundle_id is added to the
+  `dcp-query-data-input-queue-*` which eventually calls the `query-load-data-*` lambda which calls an extract transform 
```

Comment:
couldn't `*` be templated as `[deployment-stage]`?

Reply:
yes